prow/scripts/lib/gcloud.sh

```diff
@@ -317,13 +317,19 @@ function gcloud::delete_network {
 #
 # Arguments:
 # $1 - cluster name
+# $2 - optional additional labels for the cluster
 function gcloud::provision_gke_cluster {
   if [ -z "$1" ]; then
     log::error "Cluster name not provided. Provide proper cluster name."
     exit 1
   fi
   CLUSTER_NAME=$1
 
+  if [ -z "$2" ]; then
+    log::info "No additional labels provided"
+  fi
+  ADDITIONAL_LABELS=$2
```

Comment:
I think output here is irrelevant. If the 2nd argument is not provided then just continuing without any message is much more convenient for optional parameters. It can be simplified to just `ADDITIONAL_LABELS=$2`. It's a cosseting change and can be ignored. It's just a suggestion.

Reply:
`log::info` was removed